ntoskrnl/inbv/inbv.c

```diff
@@ -1279,7 +1279,7 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
 #endif
 
 #ifdef INBV_ROTBAR_IMPLEMENTED
-        if (Bar)
+        if (ROT_BAR_DEFAULT_MODE == RB_SQUARE_CELLS && Bar)
```

Comment:
```suggestion
        if ((ROT_BAR_DEFAULT_MODE == RB_SQUARE_CELLS) && Bar)
```

Reply:
TempRotBarSelection variable should have the value

Reply:
My mistake. Should I use additional brackets with TempRotBarSelection too?

Reply:
code style says preferably yes.